src/Sylius/Component/Payment/Model/PaymentMethodInterface.php

```diff
@@ -47,6 +47,10 @@ public function getPosition(): ?int;
 
     public function setPosition(?int $position): void;
 
+    public function setGatewayConfig(?GatewayConfigInterface $gatewayConfig): void;
+
+    public function getGatewayConfig(): ?GatewayConfigInterface;
+
```

Comment:
getter goes first usually. (for the rest of the codebase)

```suggestion
    public function getGatewayConfig(): ?GatewayConfigInterface;
    
    public function setGatewayConfig(?GatewayConfigInterface $gatewayConfig): void;
    
```